TradeItIosTicketSDK2.xcodeproj/project.pbxproj

```diff
@@ -2618,6 +2683,21 @@
 			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-ExampleAppUITests/Pods-ExampleAppUITests-frameworks.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
+		DCADBA6D6AC9729A75E4BCA8 /* [CP] Copy Pods Resources */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			name = "[CP] Copy Pods Resources";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-ExampleAppUITests/Pods-ExampleAppUITests-resources.sh\"\n";
+			showEnvVarsInLog = 0;
+		};
```

Comment:
Why did this change? Did you run `pod install` with the latest version of cocoapods?

Reply:
Au contraire, an old version of cocopods I believe (`1.4.x`). This was run on the pair station. What's the move, re-run `pod install` with an up to date cocopods?

Reply:
Yes please